### DISCUSSION TITLE
How to increase the speed at which HFSS executes PyAEDT code？

### DISCUSSION THREAD

**Discussion by Dicksky** [ansys/pyaedt#3212] [Q&A] [answered]
Hello, I am trying to build 10k rectangles in HFSS to do some weird manual material phase meshing, and I just find that HFSS can run the
rectangles generation code very fast at the beginning and after some time it starts to become slower, and then slower and slower. Why does the system move like that? Is there any way to avoid the running speed decrease of the code?

**Comment by Samuelopez-ansys** [comment, marked as answer]
@Dicksky You could try to first uncheck the autosave
![image](https://github.com/ansys/pyaedt/assets/85613111/afe1a8cc-9380-4752-bd6d-562bed50689e)
 aedtapp.autosave_disable()
